renderer/tr_light.cpp

```diff
@@ -1548,10 +1565,10 @@ void R_AddSingleModel( viewEntity_t* vEntity ) {
 	if ( !vEntity->scissorRect.IsEmpty() || R_HasVisibleShadows( vEntity ) ) {
 		model = R_EntityDefDynamicModel( &def );
 		if ( model == NULL || model->NumSurfaces() <= 0 ) {
-			if ( def.parms.timeGroup ) {
+			/*if ( def.parms.timeGroup ) {
```

Comment:
Ah, interesting.
Yeah, I combed our code base and am reasonably certain that it's not used and should probably be deleted - if we ever do need it, it would probably require some significant overhaul, not just to make it thread-safe.